node/src/ops/ReduceMean.h

```diff
@@ -0,0 +1,15 @@
+#ifndef NODE_OPS_ReduceMean_H_
```

Comment:
`NODE_OPS_ReduceMean_H_ => NODE_OPS_REDUCEMEAN_H_`  for L.1, L.2 and L.15

Reply:
Solved